requirements/test.txt

```diff
@@ -10,4 +10,4 @@ aiohttp
 Pillow
 requests
 flask
-pydantic>=1.0.0,<3.0
+pydantic>=2.0.0,<3.0
```

Comment:
We want to make sure to run the tests using pydantic v2 or the new implementation tests will fail